msprime/mutations.py

```diff
@@ -83,14 +83,95 @@ class JukesCantor(MutationModel):
     """
 
     def __init__(self):
-        alleles = [b"A", b"C", b"T", b"G"]
+        alleles = [b"A", b"C", b"G", b"T"]
```

Comment:
Gee, we should have these defined in just one place, to avoid errors like this?  At least, just in this file as `_ACGT_ALLELES`?